libs/auth/state/src/effects/auth.effects.ts

```diff
@@ -127,11 +125,11 @@ export class DaffAuthEffects {
       DaffAuthLoginActionTypes.LogoutSuccessAction,
     ),
     filter((action) => {
-      // if the auth check failure is for any reason other than auth failure, don't reset
+      // if the auth check failure is for any reason other than auth failure,
+      // such as a network failure, don't reset
       if (
         action.type === DaffAuthActionTypes.AuthCheckFailureAction
-        && action.errorMessage.code !== DaffAuthDriverErrorCodes.UNAUTHORIZED
-        && action.errorMessage.code !== DaffAuthDriverErrorCodes.AUTHENTICATION_FAILED
+          && !DAFF_AUTH_UNAUTHENTICATED_ERROR_CODES.find((code) => code === action.errorMessage.code)
```

Comment:
I think these look-ups should be replaced with dictionaries instead. O(1) vs. O(3).